src/init/mod.rs

```diff
@@ -46,6 +46,10 @@ impl StarshipPath {
             .map(|s| s.replace("'", "''"))
             .map(|s| format!("'{}'", s))
     }
+    /// Cmd specific path escaping
+    fn sprint_cmd(&self) -> io::Result<String> {
```

Comment:
Can we call the shell `cmdexe` or something similar when it's not obvious that it's a shell from context? (e.g. the addition of `Shell::Cmd` is fine because it's obviously a shell in that context). I'm afraid in two years, I'm going to be reading this code and get confused by `cmd` as an abbreviation for "Command" and `cmd` meaning the shell.